src/components/plots/PlotContainer.jsx

```diff
@@ -84,12 +109,40 @@ const PlotContainer = (props) => {
     </Space>
   );
 
+  const TILE_MAP = {
+    [PLOT]: {
+      toolbarControls: renderPlotToolbarControls(),
+      component: () => <center>{children}</center>,
```

Comment:
Do we need this `center` here? Also I read that center is removed from HTML5 and instead you should use `text-align` css property - https://www.techonthenet.com/html/elements/center_tag.php#:~:text=Description,as%20the%20element.

Reply:
Replaced it with a <div>